api/model.go

```diff
@@ -1369,13 +1371,18 @@ type Server struct {
 	m   *manager.VolumeManager
 	wsc *controller.WebsocketController
 	fwd *Fwd
+
+	httpClient *http.Client
 }
 
 func NewServer(m *manager.VolumeManager, wsc *controller.WebsocketController) *Server {
 	s := &Server{
 		m:   m,
 		wsc: wsc,
 		fwd: NewFwd(m),
+		httpClient: &http.Client{
+			Timeout: 30 * time.Second,
```

Comment:
Does it mean support bundle only has 30 seconds to generate? generation a big support bundle can take hours. It cannot be done in a synchronized manner.

Reply:
This 30 second is the time for longhorn to download the support bundle zip file from the support bundle manager, which gets deployed by the deployment. So longhorn has a server to handle a lot of requests, and it uses this client to download the zip file from the support bundle manager which itself has a server running inside. @yasker

Reply:
Does that mean the support bundle needs to be downloaded in 30 seconds? And why do we need to define this one-time HTTP client in the server object?